src/components/containers/ImageAccordion.js

```diff
@@ -13,8 +13,8 @@ class ImageAccordion extends Component {
 
     const content =
       images.length &&
-      images.map((ann, i) => (
-        <ImageFold key={i} imageIndex={i} name={ann.text} canDelete={canAdd}>
+      images.map((img, i) => (
+        <ImageFold key={i} imageIndex={i} name={img.text} canDelete={canAdd}>
```

Comment:
images don't have `.text` do they? Perhaps make a name from `.source`? Not sure what to do about shapes... index and type perhaps?

Reply:
hehe so this change is @VeraZab cleaning up my copy-paste work from a previous PR... When I created the Images panel I'd based it off of the Notes panel but not renamed `ann` to `img` (and ditto Shapes). So any issues with this code should probably go in a separate issue that I can handle ;)